src/components/AlertBase.tsx

```diff
@@ -12,7 +12,15 @@ const AlertBase = (
   { severity = 'success', variant = 'standard', ...props }: AlertBaseProps,
   ref: Ref<HTMLDivElement>
 ): JSX.Element => {
-  return <MuiAlert ref={ref} severity={severity} variant={variant} {...props} />;
+  return (
+    <MuiAlert
+      ref={ref}
+      severity={severity}
+      variant={variant}
+      sx={{ alignItems: 'center', '& .MuiAlert-action': { pt: 0, pl: 5 } }}
```

Comment:
I would destructure sx and put it here to avoid overriding default sx props accidentally :)
```suggestion
      sx={{ alignItems: 'center', '& .MuiAlert-action': { pt: 0, pl: 5 } }}
```